app/serializers/transaction_serializer.rb

```diff
@@ -1,5 +1,3 @@
 class TransactionSerializer < ActiveModel::Serializer
   attributes :id, :invoice_id, :credit_card_number, :result
-
-  belongs_to :invoice
```

Comment:
This line was causing the customer/transactions spec harness to fail. I verified that the customer record test was still able to pass without it.